test/serialisers/json-0.6.js

```diff
@@ -176,20 +176,18 @@ describe('JSON Serialiser', function() {
       var defaultElement = new minim.Element(new minim.elements.String('North'));
       defaultElement.element = 'default';
 
-      var sampleNorth = new minim.Element(new minim.elements.String('North'));
-      sampleNorth.element = 'enum';
-      var sampleEast = new minim.Element(new minim.elements.String('East'));
-      sampleEast.element = 'enum';
+      var sampleNorth = new minim.elements.Enum(new minim.elements.String('North'));
```

Comment:
As the content gets refracted (`Element.call(this, refract(content), meta, attributes);)` this could be simplified as:

```js
var sampleNorth = new minim.elements.Enum('North');
```